opentech/apply/dashboard/tests/test_views.py

```diff
@@ -61,3 +61,29 @@ def test_cannot_see_submission_in_determination_when_not_lead(self):
         ApplicationSubmissionFactory(status='concept_review_discussion', workflow_stages=2, form_data__title='Reviewr')
         response = self.get_page()
         self.assertNotContains(response, 'Reviewr')
+
+    def test_waiting_for_review_with_count(self):
+        submission = ApplicationSubmissionFactory(status='external_review', workflow_stages=2, reviewers=[self.user])
+        response = self.get_page()
+        self.assertContains(response, 'Waiting for your review')
+        self.assertContains(response, submission.title)
+        self.assertEquals(response.context['in_review_count'], 1)
+
+
+class TestReviewerDashboard(BaseViewTestCase):
+    user_factory = ReviewerFactory
+    url_name = 'dashboard:{}'
+    base_view_name = 'dashboard'
+
+    def test_waiting_for_review_with_count(self):
+        submission = ApplicationSubmissionFactory(status='external_review', workflow_stages=2, reviewers=[self.user])
+        response = self.get_page()
+        self.assertContains(response, 'Waiting for your review')
+        self.assertContains(response, submission.title)
+        self.assertEquals(response.context['in_review_count'], 1)
+
+    def test_no_submissions_waiting_for_review(self):
+        submission = ApplicationSubmissionFactory(status='external_review', workflow_stages=2, reviewers=[])
```

Comment:
need 1 more test to confirm they dont get submissions not in the external review phase.

Reply:
@emullaney I'm not sure i can see a test case where the user is assigned as a reviewer but it isnt in the external phase.